docs/src/components/ColorSwatch/ColorSwatch.component.tsx

```diff
@@ -0,0 +1,110 @@
+// REACT
+import React, { useState } from 'react';
+// VENDOR
+import styled from 'styled-components';
+import { ColorHelpers } from '../ColorHelpers';
+import { Pill } from '../Pill'
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import { Typography } from '../../../../src/Typography';
+// THEME
+import { colors } from '../../../../src/theme';
+
+
+// INTERFACES
+interface ColorSwatchProps {
+  colorHexCode: string;
+  colorName: string;
+}
+
+
+// STYLES
+const StyledColorSwatch = styled.div`
+  border: 0.5rem solid ${props => props.highContrastBorderColor}};
+  display: inline-block;
+  margin: 0 2rem 2rem 0;
+  width: 17rem;
+
+  .color-swatch-pill-container {
+    background: ${props => props.colorHexCode };
+    cursor: pointer;
+    padding: 7rem 1rem 1rem 1.5rem;
+  }
+
+  .color-swatch-text-container {
+    border-top: 0.05rem solid ${props => props.highContrastBorderColor};
+    padding: 1rem;
+  }
+
+  .color-swatch-text-container p {
+    margin: 0;
+    padding: 0.25rem 0 0.25rem 0.5rem;
+  }
+
+  .color-swatch-text-container p:hover {
```

Comment:
This can be nested very much the same way:

```
.color-swatch-text-container p:hover {}
```
can be
```
.color-swatch-text-container {
    p {
        &:hover {}
    }
}
```

Reply:
Just remember:
> Note that objects should have at most 4 levels. Most of the time you'll stay around the 2-3 level range, and the fourth would be the interaction state.
~[Sauce](http://thesassway.com/beginner/the-inception-rule)

Reply:
Will do. I've tasted this sauce before. It takes me back to Tuscany. 🍷